dependencies.props

```diff
@@ -36,7 +36,7 @@
     <PgoDataPackageVersion>99.99.99-master-20180703-0030</PgoDataPackageVersion>
     <MicrosoftNETCoreRuntimeCoreCLRPackageVersion>3.0.0-preview1-26704-01</MicrosoftNETCoreRuntimeCoreCLRPackageVersion>
     <XunitPackageVersion>2.2.0-beta2-build3300</XunitPackageVersion>
-    <XunitConsoleNetcorePackageVersion>2.2.0-preview1-02830-02</XunitConsoleNetcorePackageVersion>
+    <XunitPackageVersion>2.4.0-beta.2.build4010</XunitPackageVersion>
```

Comment:
Should remove one of these 😄

Reply:
Oops, this happened in my branch when I rebased. Thanks!

Reply:
Sorry, I should have made that more clear. I merged @sbomer 's branch into this one, as we've both changed a bunch of stuff. I wonder if it's worth waiting for Sven's changes to land, then rebase onto that, so this is clearer?

Reply:
I think that would make sense.